applications/StructuralMechanicsApplication/tests/mesh_moving_test/simple_mesh_moving_test_parameters.json

```diff
@@ -15,10 +15,16 @@
         "echo_level"                         : 0,
         "domain_size"     : 2,
         "time_stepping"                      : {
-            "time_step" : 0.005
-        },
-        "linear_solver_settings" : {
-            "solver_type" : "ExternalSolversApplication.super_lu"
+            "time_step_intervals" : {
+                "first_interval" : {
```

Comment:
this should be a list if you don't need the key

Reply:
I feel more flexible that way, so you can use names to identify the intervals

Reply:
It is for practical reasons

Reply:
And it is not a list if we want to keep the Parameters interface